### PR TITLE
Module bundles

### PR DESCRIPTION
**Pull request by tarteo**
* Removed old installation modules
* Refactor base_onboarding to make it easier extendable
* Module bundles (base_module_bundle)
* Installation of modules (bundles) at onboarding
* New installation modules (bundles): account_install, website_install, website_sale_install, membership_install, crm_install, event_install, hr_install, mass_mailing_install, project_install, website_event_install, sale_install
* Remove options that install modules from res.config.settings
* Backwards compatibility with container_install_basis

image

### CODE REVIEW

**Comment by tarteo** [comment]
Also added a helpdesk bundle

File: container_install/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+        "spreadsheet_oca_ux",
+        "partner_country_default_nl",
+        "remove_odoo_enterprise",
+        "base_import_debranding",
+        "digest_disable",
+        "partner_external_map",
+        "fetchmail_notify_error_to_sender",
+        "base_municipality",
+        "product_category_product_link",
+        "base_module_bundle",
```
Comment (ByteMeAsap) [inline container_install/__manifest__.py:46]: should mail_debrand module be a part of this?

File: mass_mailing_install/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,11 @@
+{
+    "name": "Email Marketing",
+    "summary": "Design, send and track emails",
+    "author": "Onestein",
+    "website": "https://onestein.nl",
+    "category": "Email Marketing",
+    "version": "16.0.1.0.0",
+    "license": "AGPL-3",
+    "depends": ["mass_mailing", "mass_mailing_force_dedicated_server"],
```
Comment (ByteMeAsap) [inline mass_mailing_install/__manifest__.py]: Should mass_mailing_partner be a part of this?

File: membership_install/__manifest__.py [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+    "version": "16.0.1.0.0",
+    "license": "AGPL-3",
+    "depends": [
+        "membership",
+        "membership_accessibility",
+        "membership_committee",
+        "membership_section",
+        "membership_contribution",
+        "membership_extension",
+        "membership_origin",
```
Comment (ByteMeAsap) [inline membership_install/__manifest__.py]: We are going to remove this module as well as membership_report and add tags on members, so this should be excluded I think or we take care of this when membership changes are merged?

File: project_install/__manifest__.py [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+    "website": "https://onestein.nl",
+    "category": "Project",
+    "version": "16.0.1.0.0",
+    "license": "AGPL-3",
+    "depends": [
+        "project",
+        "project_parent",
+        "project_timeline",
+        "project_task_add_very_high",
+        "project_scrum",
```
Comment (ByteMeAsap) [inline project_install/__manifest__.py:14]: I believe project_list,project_merge,project_task_default_stage,project_task_link might be useful and good to have for most customers

**Review by ByteMeAsap — COMMENTED** [review]
Fantastic work man!!! Just a few comments on the modules being considered

**Review by andreschenkels — COMMENTED** [review]
I would rcommend these modules:
mass_mailing_custom_unsubscribe

And this one if not already in the base container.
mail_partner_opt_out

**Review by andreschenkels — APPROVED** [review]

**Comment by andreschenkels** [comment]
@tarteo project_scrum needs to be added

**Pull request merged**